README.md

```diff
@@ -66,10 +70,37 @@ This will install the following files (e.g. on Ubuntu):
 
 For iHD_drv_video.so please export related LIBVA environment variables.
 ```
-export LIBVA_DRIVERS_PATH=<path-contains-iHD_drv_video.so>
+export LIBVA_DRIVERS_PATH=/usr/lib/x86_64-linux-gnu/dri:/usr/lib/dri:<path-contains-iHD_drv_video.so>
```

Comment:
We can't list all possible examples here. For example, you miss Fedora flavored systems. Please, revert.

Reply:
Will do. Ideally a user already knows where their expected dri path is, and the installation path from cmake will be the same